### PR TITLE
Add document to setup SwiftLint for iOS swift apps

### PR DESCRIPTION
**Pull request by anand00**
- Setup ``SwiftLint`` with basic iOS app in Swift.
- Considered Cocoapods for ``SwiftLint`` installations
- Used rules based on Raywenderlich Swift Style Guide 
- This configuration is tested on Swift 5.6.

### CODE REVIEW

File: swift-swiftLint.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Introduction
+
+[SwiftLint](https://github.com/realm/SwiftLint/blob/master/README.md) is a tool by Realm to enforce Swift style and conventions. SwiftLint hooks into Apple’s own SourceKit framework to parse your Swift code, which means it supports the full range of Swift syntax and remains up to date as Swift continues to evolve.
+<br/>It is based on guidelines from the Swift style guide.
```
Comment (Abijeet) [inline swift-swiftLint.md]: I would skip the `<br/>` tags and instead add just empty lines.

File: swift-swiftLint.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  min_length: 3 # only warning
+  max_length: # warning and error
+    warning: 30
+    error: 35
+  excluded: iPhone # excluded via string
+reporter: "xcode" # reporter type (xcode, json, csv, checkstyle, codeclimate, junit, html, emoji, sonarqube, markdown, github-actions-logging)
+```
+
+# Usage
+If you've installed SwiftLint via CocoaPods, then 1. **Select Target** -> 2. **Build Phase** -> 3. **Click on +** -> 4. **Add New Run Script Phase** and <br /> copy & paste below script in Run Script
```
Comment (Abijeet) [inline swift-swiftLint.md]: Can we put the items 1, 2, 3, in new lines?

File: swift-swiftLint.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+```bash
+"${PODS_ROOT}/SwiftLint/swiftlint"
+```
+<img width="1200" alt="build-phaseimg" src="https://user-images.githubusercontent.com/6582396/166877073-ec3594d3-c58b-4b4c-89f8-86529092cab8.png">
```
Comment (Abijeet) [inline swift-swiftLint.md]: This image should be adding in the current repository itself

File: swift-swiftLint.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+
+* Move the ``Run Script`` above the ``Compile Sources`` step by drag and drop, to detect errors quickly before compiling.
```
Comment (Abijeet) [inline swift-swiftLint.md]: Use single tilde `

**Review by Abijeet — CHANGES_REQUESTED** [review]

File: swift-swiftLint.md (hunk 1 above)
Comment (anand00) [reply swift-swiftLint.md]: updated

File: swift-swiftLint.md (hunk 2 above)
Comment (anand00) [reply swift-swiftLint.md]: updated

File: swift-swiftLint.md (hunk 3 above)
Comment (anand00) [reply swift-swiftLint.md]: updated image URL

File: swift-swiftLint.md (hunk 4 above)
Comment (anand00) [reply swift-swiftLint.md]: updated

File: swift-swiftLint.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
+# Introduction
```
Comment (vishnu-kyatannawar) [inline swift-swiftLint.md]: Please create a `assets` folder and keep images there.

File: swift-swiftLint.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+[SwiftLint](https://github.com/realm/SwiftLint/blob/master/README.md) is a tool by Realm to enforce Swift style and conventions. SwiftLint hooks into Apple’s own SourceKit framework to parse your Swift code, which means it supports the full range of Swift syntax and remains up to date as Swift continues to evolve. It is based on guidelines from the Swift style guide.
```
Comment (vishnu-kyatannawar) [inline swift-swiftLint.md]: We need to add swift from TODO to Technologies list in README.md.

image

Comment (vishnu-kyatannawar) [reply swift-swiftLint.md]: Please ensure to add version number like other technologies.

File: swift-swiftLint.md (hunk 5 above)
Comment (vishnu-kyatannawar) [inline swift-swiftLint.md]: Please, rename this file name to `swift-swiftlint.md` (all small case). We can standadize a convention for file name later on.

File: swift-swiftLint.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+# Installation
+
+### Using [Homebrew](http://brew.sh/):
+
+```
+brew install swiftlint
+```
+
+### Using [CocoaPods](https://cocoapods.org):
```
Comment (vishnu-kyatannawar) [inline swift-swiftLint.md]: We should standadize to use 1 thing. Either `homebrew` or `cocapods`.

If they are like `npm` and `yarn`. We decided to go with just `npm`, then you need to pick the good and reliable one and only use that.

**Review by vishnu-kyatannawar — CHANGES_REQUESTED** [review]

File: swift-swiftLint.md (hunk 5 above)
Comment (anand00) [reply swift-swiftLint.md]: created assets folder and moved images

File: swift-swiftLint.md (hunk 6 above)
Comment (anand00) [reply swift-swiftLint.md]: updated

File: swift-swiftLint.md (hunk 5 above)
Comment (anand00) [reply swift-swiftLint.md]: file name updated

File: swift-swiftLint.md (hunk 7 above)
Comment (anand00) [reply swift-swiftLint.md]: - Removed Carthage option.
- Recommended CocoaPods

**Review by vishnu-kyatannawar — APPROVED** [review]

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**